www/js/vis.js

```diff
@@ -2374,7 +2374,12 @@ var vis = {
                                 if (value === undefined || value === null) {
                                     value = this.states.attr(oids[t].operations[k].arg[a].visOid);
                                 }
-                                string += 'var ' + oids[t].operations[k].arg[a].name + ' = "' + value + '";';
+                                try {
+                                    if (JSON.parse(value));
+                                    string += 'var ' + oids[t].operations[k].arg[a].name + ' = ' + value + ';';
+                                } catch(e) {
```

Comment:
For me the following worked much better: 

`                                try {
                                    value = JSON.parse(value);
                                    string += 'var ' + oids[t].operations[k].arg[a].name + ' = JSON.parse("' + JSON.stringify(value).replace(/\x22/g, '\\\x22') + '");';
                                } catch(e) {
                                    string += 'var ' + oids[t].operations[k].arg[a].name + ' = "' + value + '";';
                                }
`